lib/matplotlib/transforms.py

```diff
@@ -35,24 +35,16 @@
 from matplotlib.externals import six
 
 import numpy as np
-from numpy import ma
 from matplotlib._path import (affine_transform, count_bboxes_overlapping_bbox,
     update_path_extents)
 from numpy.linalg import inv
 
 import weakref
 import warnings
-try:
-    set
-except NameError:
-    from sets import Set as set
 
 from .path import Path
 
 DEBUG = False
-# we need this later, but this is very expensive to set up
-MINFLOAT = np.MachAr(float).xmin
```

Comment:
I am also mildly concerned about this, but this is almost certainly not the bottle neck any more.